custom_tracing/src/lib.rs

```diff
@@ -54,7 +55,7 @@ where
 }
 
 pub fn record_b58(key: &str, pub_key: &[u8]) {
-    let b58 = bs58::encode(pub_key).into_string();
+    let b58 = PublicKeyBinary::from(pub_key).to_string();
```

Comment:
huh? what format was that key string thing there?

Reply:
Just wanted to print b58 my old stuff was bad did not realized that `to_string` on a `PublicKeyBinary` or `PublicKey` gives you the b58 string version